src/test/java/uk/gov/dvsa/mot/fixtures/mot/PurchaseHistoryPageStepDefinitions.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.dvsa.mot.fixtures.mot;
+
+import static junit.framework.TestCase.assertTrue;
+
+import cucumber.api.java8.En;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.dvsa.mot.framework.WebDriverWrapper;
+
+import javax.inject.Inject;
+
+/**
+ * Step definitions specific to the purchase history page of the mot app.
+ */
+public class PurchaseHistoryPageStepDefinitions implements En {
+
+    /** The logger to use. */
+    private static final Logger logger = LoggerFactory.getLogger(HomePageStepDefinitions.class);
```

Comment:
The logger should create an instance with the current class name i.e. CPMSStepDefinitions.class so that logs are recorded against the correct class.